tests/api2/test_480_system_advanced.py

```diff
@@ -18,88 +17,84 @@ def sysadv_dict():
     return {}
 
 
-def test_01_system_advanced_get():
-    results = GET('/system/advanced/')
-    assert results.status_code == 200, results.text
-    assert isinstance(results.json(), dict)
+def test_system_advanced_get():
+    results = call('system.advanced.config')
```

Comment:
I think this test can be totally removed. It is pointless and other tests will fail spectacularly if we stop returning dict.